runtime/vm/jvminit.c

```diff
@@ -1822,6 +1826,45 @@ processMemoryInterleaveOptions(J9JavaVM * vm)
 	j9port_control(J9PORT_CTLDATA_VMEM_NUMA_INTERLEAVE_MEM, enabled? 1 : 0);
 }
 
+#if JAVA_SPEC_VERSION >= 19
+/**
+ * -XX:ContinuationCache:t1=<U_32>,t2=<U_32>
```

Comment:
Is `-XX:ContinuationCache:t1=<U_32>` acceptable, i.e., can t2 be a default value?

Reply:
I can make the parsing to allow this case. Is this needed?

Reply:
As discussed with Jack offline, the user is expected to provide both `t1` and `t2` values.